feat: Subject Model, Route, Controller

LucasDLabo opened the pull request:
Se creo la asignación masiva de Subject. Se creo la respectiva Route Resource con su Controller

LucasDLabo committed:
feat: Subject Model, Route, Controller

Se creo la asignación masiva de Subject. Se creo la respectiva Route Resource con su Controller

MaksiConGit commented:
Te falta el destroy():
$registro = TuModelo::find($id);   
$registro->is_deleted = true;
$registro->save();
return redirect()->route('ruta.deseada');

LucasDLabo committed:
feat: CRUD Subject

Se realizo el CRUD de Subject con las correspondientes validaciones y mensajes de error. Se añadió el atributo "is_deleted" a la propiedad $fillable para su correcto funcionamiento. Se cambio el atributo authorize() dentro de la request a "true" para su correcto funcionamiento. Se hizo que los registros que son eliminados se muestre con una anotación para mejor organización

LucasDLabo committed:
Merge branch 'code' into 110-subject-model-route-controller
